Px.Utils/Validation/ContentValidation/ContentValidationFunctions.cs

```diff
@@ -0,0 +1,973 @@
+﻿using PxUtils.PxFile;
+using PxUtils.Validation.SyntaxValidation;
+
+namespace PxUtils.Validation.ContentValidation
+{
+    public delegate ValidationFeedbackItem[]? ContentValidationEntryFunctionDelegate(ValidationStructuredEntry entry, PxFileSyntaxConf syntaxConf, ref ContentValidationInfo info);
```

Comment:
Delegates are functions by definition, no need to name them as such. ContentEntryValidationDelegate is enough.